packages/contracts-bedrock/semver-lock.json

```diff
@@ -1,14 +1,14 @@
 {
   "src/EAS/EAS.sol": "0xea04b7a9ec170ce9cbd466ede87650f9e9ffe3d725d7951cef5777a98a840173",
   "src/EAS/SchemaRegistry.sol": "0x5ee1a0c3b2bf1eb5edb53fb0967cf13856be546f0f16fe7acdc3e4f286db6831",
-  "src/L1/DelayedVetoable.sol": "0x276c6276292095e6aa37a70008cf4e0d1cbcc020dbc9107459bbc72ab5ed744f",
+  "src/L1/DelayedVetoable.sol": "0xc59a6b5e7fe976000d6e9a702d97f048b512938a2ff6dc17f32678ab50da73f3",
```

Comment:
Does this need a rebase? Why would the semver lock be changing here?

Reply:
Rebased